lib/fog/compute/kubevirt/models/node.rb

```diff
@@ -0,0 +1,39 @@
+module Fog
+  module Compute
+    class Kubevirt
+      class Node < Fog::Model
+        identity :name
+
+        attribute :namespace,        :aliases => 'metadata_namespace'
+        attribute :resource_version, :aliases => 'metadata_resource_version'
+        attribute :uid,              :aliases => 'metadata_uid'
+        attribute :os_image,         :aliases => 'status_node_info_os_image'
+        attribute :operating_system, :aliases => 'status_node_info_operating_system'
+        attribute :kernel_version,   :aliases => 'status_node_info_kernel_version'
+        attribute :hostname,         :aliases => 'status_addresses_hostname'
+        attribute :ip_address,       :aliases => 'status_addresses_ip'
+
+        def self.parse(object)
+          metadata = object[:metadata]
+          status = object[:status]
+          info = status[:nodeInfo]
+
+          addresses = status[:addresses]
+          hostname = addresses.detect { |address| address[:type] == 'Hostname' }[:address]
```

Comment:
can through an exception if detect return nil, next line as well.

Reply:
We always have an address there so `nil` is not an option here.